docs/tutorial/get-started.md

```diff
@@ -634,6 +657,44 @@ This chapter uses [NGT](https://github.com/yahoojapan/ngt) as Vald Agent to perf
 
           </details>
 
+   1. Remove
+
+      - Remove indexed 400 training datasets from the Vald agent.
+          <details><summary>example code</summary><br>
+
+        ```go
+        for i := range ids [:insertCount] {
+            _, err := client.Remove(ctx, &payload.Remove_Request{
+                Id: &payload.Object_ID{
+                    Id: ids[i],
+                },
+            })
+            if err != nil {
+                glg.Fatal(err)
+            }
+            if i%10 == 0 {
+                glg.Infof("Removed %d", i)
+            }
+        }
+        ```
+
+          </details>
+
+
+      - Remove manually instead of waiting for auto indexing.
+        The removed vectors are indexed in NGT graph yet before auto indexing even you called Remove API before.
+        If you run the below code, the indexes will be removed completely from the Vald Agent NGT graph and the Backup file.
```

Comment:
```suggestion
      - Remove from the NGT graph manually instead of waiting for auto indexing.
        The removed vectors are still exists in the NGT graph before the SaveIndex (or CreateAndSaveIndex) API is called.
        If you run the below code, the indexes will be removed completely from the Vald Agent NGT graph and the Backup file.
```

Reply:
```suggestion
      - Remove from the index manually instead of waiting for auto indexing.
        The removed vectors are still exist in the NGT graph index before the SaveIndex (or CreateAndSaveIndex) API is called.
        If you run the below code, the indexes will be removed completely from the Vald Agent NGT graph and the Backup file.
```